svc-lighthouse-api/src/main/java/gov/va/vro/abddataaccess/service/FhirClient.java

```diff
@@ -319,14 +319,14 @@ private List<BundleEntryComponent> getRecords(
   }
 
   private synchronized String getLastUpdated() {
-    if (lastUpdated == null) {
-      lastUpdated = "";
+    if (lastUpdated.isEmpty()) {
       String lastUpdatedDate = properties.getFilterLastUpdatedDate();
       if ((lastUpdatedDate != null) && !lastUpdatedDate.isEmpty()) {
         log.info("Update lastUpdated to {}", lastUpdatedDate);
         SimpleDateFormat dft = new SimpleDateFormat(LAST_UPDATED_DATE_FORMAT);
         try {
-          Date date = dft.parse(lastUpdatedDate);
+          // Verify that it is in the expected date format
+          dft.parse(lastUpdatedDate);
```

Comment:
The `date` variable wasn't being used so I assume the `dft.parse(lastUpdatedDate)` was simply to check that `lastUpdatedDate` is in the expected date format. Is my assumption correct?